hasura/graphql-engine
# console: handle dbs with no public schema

Solves #2652 

### Changelog


### Affected components


### Solution and Design

* Fetch all schemas names,
* If there's no `public` schema, then set `currentSchema` as the first one from the list,
* If there's no `public` schema, I'm showing info message `No public schema` (to be discussed).

### Steps to test and verify
Open the console with no `public` schema. Click on `data`. You should be redirected to the first schema from the list.
